packages/react-app/src/lib/queryClient.ts

```diff
@@ -12,20 +12,27 @@ const queryClient = new QueryClient({
   },
 });
 
-const useGiveaways = () => useQuery('active', API.getGiveaways);
+const useGiveaways = () =>
+  useQuery('active', API.getGiveaways, { refetchOnMount: 'always' });
```

Comment:
Is there a reason we need to re-fetch always and not only when the data is stale?

Reply:
In those cases we should use react query's refetch functionally to refetch only when needed so we can take advantage of the cache functionalities and reduce the ammount of unecessary requests: https://tanstack.com/query/v4/docs/react/reference/QueryClient#queryclientfetchquery

It will look something like this:
```
 const { mutate } = useMutation(editGiveaway, {
    onSuccess: () => queryClient.refetchQueries(['giveaways'])
 })
```